runtime/onert/backend/trix/DevContext.cc

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
For now, things related to the device handle are gathered here, but when implementing npu daemon, others except the context roles should be separated.